src/aiperf/dataset/loader/sagemaker_data_capture.py

```
@@ -0,0 +1,232 @@
+            elif path.is_file():
+                target = path
+            else:
+                return False
+            try:
+                with open(target) as f:
+                    for line in f:
+                        if not (line := line.strip()):
+                            continue
+                        record = orjson.loads(line)
+                        return "captureData" in record and "eventMetadata" in record
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)